<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaybackSettingsClass</class>
 <widget class="QDialog" name="PlaybackSettingsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>210</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>310</width>
    <height>210</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>310</width>
    <height>210</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Algorithm Playback</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Playback Settings</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>291</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="0">
     <widget class="QCheckBox" name="showPseudocode">
      <property name="text">
       <string>Show Pseudocode</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Step delay in milliseconds:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Iterations per step:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QCheckBox" name="startPaused">
      <property name="text">
       <string>Start Paused</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QSpinBox" name="iterationsPerStep">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100000</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QSpinBox" name="stepDelay">
      <property name="minimum">
       <number>500</number>
      </property>
      <property name="maximum">
       <number>30000</number>
      </property>
      <property name="singleStep">
       <number>500</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="startPlayback">
   <property name="geometry">
    <rect>
     <x>199</x>
     <y>180</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Playback</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runInstantly">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Run Instantly</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>stepDelay</tabstop>
  <tabstop>iterationsPerStep</tabstop>
  <tabstop>showPseudocode</tabstop>
  <tabstop>startPaused</tabstop>
  <tabstop>startPlayback</tabstop>
  <tabstop>runInstantly</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
